composer.json

```diff
@@ -5,19 +5,19 @@
     "license": "MIT",
     "minimum-stability": "dev",
     "require": {
-        "php": ">=8.1",
+        "php": ">=8.3",
         "ext-json": "*",
-        "symfony/config": "^5.4 | ^6.0 | ^7.0",
-        "symfony/dependency-injection": "^5.4 | ^6.0 | ^7.0",
-        "symfony/http-kernel": "^5.4 | ^6.0 | ^7.0",
-        "symfony/routing": "^5.4 | ^6.0 | ^7.0",
-        "symfony/deprecation-contracts": "^2.2 | ^3.0"
+        "symfony/config": "^6.4.5 | ^7.0",
+        "symfony/dependency-injection": "^6.4.5 | ^7.0",
+        "symfony/http-kernel": "^6.4.5 | ^7.0",
+        "symfony/routing": "^6.4.5 | ^7.0",
+        "symfony/deprecation-contracts": "^3.0"
```

Comment:
It would be more friendly... and probably ok for us, to still allow v2 of the contract